### PR TITLE
[NON-MODULAR] More drugs. (Part 2)

### PR DESCRIPTION
**Pull request by linnpap**
## About The Pull Request
image
Adds some extra drugs, such as THC, various forms of cannabis, and PCP.

## Why It's Good For The Game
Makes weed no longer as boring and adds RP through the existence of these drugs.

## Changelog
:cl:
add: Added THC
add: Added hash (along with hash bricks)
add: Added dabs
add: Added PCP
add: Added more distinct effects to quaaludes and opium/heroin
balance: Cannabis no longer produces space drugs, it now produces THC
balance: You can now no longer produce as many drugs
fix: Fixed the ability to equip opium poppies as a mask
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/morenarcotics/chemistry/reagents/pcp.dm [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	description = "Pure rage put into chemical form."
+	reagent_state = LIQUID
+	color = "#ffea2e"
+	overdose_threshold = 10 //really low overdose to keep people from abusing it too much
+	ph = 8
+	taste_description = "rage"
+	var/datum/brain_trauma/special/psychotic_brawling/bath_salts/pcp_rage
+	var/datum/brain_trauma/special/tenacity/pcp_tenacity
+
+/datum/reagent/drug/pcp/on_mob_metabolize(mob/living/L)
```
Comment (ZephyrTFA) [inline modular_skyrat/modules/morenarcotics/chemistry/reagents/pcp.dm:15]: Use different variable names; its important to use clear and concise naming.

I would recommend you rename your variables to 'mob_living' and 'mob_carbon' for all of these procs.

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]

**Comment by FlamingLily** [comment]
Can I ask that the ability to sell drugs directly to central command with no further effort required such as hacking the console or some kind of black market sales pod be removed? Because too often cargo just becomes a drug empire, and it's kind of dumb, at least with our current corpregs.

Perhaps a possible change could require drugs to be packaged inconspicuously, in sealed crates.

**Comment by NullDagaf** [comment]
> Can I ask that the ability to sell drugs directly to central command with no further effort required such as hacking the console or some kind of black market sales pod be removed? Because too often cargo just becomes a drug empire, and it's kind of dumb, at least with our current corpregs.
> 
> Perhaps a possible change could require drugs to be packaged inconspicuously, in sealed crates.

Pretty sure there's already a list of blacklisted items. Don't think it'd be hard to add onto that list.

**Comment by RatFromTheJungle** [comment]
> 
> 
> Can I ask that the ability to sell drugs directly to central command with no further effort required such as hacking the console or some kind of black market sales pod be removed? Because too often cargo just becomes a drug empire, and it's kind of dumb, at least with our current corpregs.
> 
> Perhaps a possible change could require drugs to be packaged inconspicuously, in sealed crates.

the drugs are literally illegal, word-for-word from 105, and if they sell them to CC, it's a 205, while central wouldn't care because they're just after profit, if sec isn't stopping them, it's a player issue
image

**Comment by NullDagaf** [comment]
> the drugs are literally illegal, word-for-word from 105, and if they sell them to CC, it's a 205, while central wouldn't care because they're just after profit, if sec isn't stopping them, it's a player issue

Mhm.
And quick question, why should security actually care?
From how I see it, not only security has no real incentive to police these laws, but is actively discouraged from doing so - since you just get labeled as "shitsec" in-character, and a dick out-of-character.

I've witnessed in person a warden literally ignore their prisoners having meth in their possession - not even hiding it, just basically having it on the counter like it was a common thing. Heck, another officer even brought it to the warden's attention, and they just responded with "Let them have it".

Point being in all of this. If you want cargo to not ship off drugs en-masse, then actively **prevent** them from selling the drugs - either by literally not being able to ship them, or - akin to advanced magboots - making their price negative, so cargo loses credits if they decide to ship them off (with perhaps a flavor message sent to the bounty computer, notifying them that they have been "fined"). Don't rely on security to deal with this issue, because as time has proven many times over, they simply won't.

**Comment by linnpap** [comment]
i mean there are literal bounties for meth that medical can do so idk

**Review by Gandalf2k15 — APPROVED** [review]
LGTM

**Pull request merged**